components/CameraScreen.tsx

```diff
@@ -1,5 +1,9 @@
 import React, { useRef, useState } from "react";
+<<<<<<< HEAD
```

Comment:
The merge conflict markers (e.g. <<<<<<< etc.) are still in the code in a few places. Try pulling the branch and then go through and remove them. Then check if the project will still build. If it won't build, you may need to create a new branch from main and manually copy in the changes from this branch.